### DISCUSSION TITLE
Is there a way to move the taskbar to the top of the screen? I like it when the start menu is on top.

### DISCUSSION THREAD

**Discussion by swimJim** [ramensoftware/windhawk-mods#943] [Q&A]
I've been searching for a mod that lets me move the start menu or taskbar to the top of the screen like you used to be able to do on Windows 10. Is that available in Windhawk?

**Comment by m417z** [comment]
It's not available yet, but it's planned. See https://github.com/ramensoftware/windhawk-mods/issues/166.
